modules/op-creating-and-verifying-task-run-signatures-without-any-additional-authentication.adoc

```diff
@@ -33,7 +33,7 @@ $ cosign generate-key-pair k8s://openshift-pipelines/signing-secrets
 +
```

Comment:
Can't we just reference the other section here?

Reply:
We have a ban on cross-references between modules, this ban is enforced by OpenShift docs.